docs/BuildFromSource.md

```diff
@@ -233,13 +233,13 @@ To build a code change associated with a modification, run the build script in t
 On Windows, you can run the command script:
 
 ```powershell
-.\build.cmd
+.\eng\build.cmd
```

Comment:
Yeah, it's talking about individual folders and asking you to run the `build.cmd` located at their root